### PR TITLE
Fix for missing alpha on server restart

### PR DESCRIPTION
**Pull request by SamGondelman**
https://highfidelity.manuscript.com/f/cases/13860

Test plan:
- Run [this](https://gist.githubusercontent.com/SamGondelman/a94287aeb989320be13da085dd4a5cdb/raw/71fc17436c2c307a23e4bdaa08988934a37eed9c/ForwardPrimitives.js).  The second cube should be slightly transparent.
- Restart the server you're in (if you're using the server from this PR, make sure you're in localhost, not on the serverless domain).  When the entities reappear, the second cube should still be slightly transparent.

### CODE REVIEW

**Comment by hifi-gustavo** [comment]
Android build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/8715/app_12868_PR.apk

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/8715/HighFidelity-Beta-PR12868-3270409b7b2ee76f846d2eb1f2cab70f25466c28.exe

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/8715/HighFidelity-Beta-PR12868-3270409b7b2ee76f846d2eb1f2cab70f25466c28.dmg

File: libraries/entities/src/EntityItemProperties.cpp [hunk 1]
```diff
@@ -1208,6 +1208,7 @@ QScriptValue EntityItemProperties::copyToScriptValue(QScriptEngine* engine, bool
     COPY_PROPERTY_TO_QSCRIPTVALUE(PROP_ACTION_DATA, actionData);
     COPY_PROPERTY_TO_QSCRIPTVALUE(PROP_LOCKED, locked);
     COPY_PROPERTY_TO_QSCRIPTVALUE(PROP_USER_DATA, userData);
+    COPY_PROPERTY_TO_QSCRIPTVALUE(PROP_ALPHA, alpha);
```
Comment (ZappoMan) [inline libraries/entities/src/EntityItemProperties.cpp]: this is a protocol change

**Comment by Qliemillar** [comment]
Currently being tested

**Comment by Qliemillar** [comment]
Testing complete - PASS @conklin94122 
The second cube generated was transparent, and remained transparent after the domain was restarted.

**Pull request merged**